deps/wazuh_testing/wazuh_testing/modules/fim/classes.py

```diff
@@ -460,7 +460,7 @@ def validate_checkers_per_event(events, options, mode):
                 if self.is_value:
                     validate_registry_event(ev, options, mode, is_key=False)
                 else:
-                    validate_registry_event(ev, options, mode , is_key=False)
+                    validate_registry_event(ev, options, mode, is_key=False)
```

Comment:
What's the difference in this `if...else`?

Reply:
There was an error in the second line, it should be True. It was fixed in other branches. 

done in https://github.com/wazuh/wazuh-qa/pull/4009/commits/2c11167dfbd10f2e4f190449c19831b52cde646a